src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -1,13 +1,15 @@
-<div id="sfwdt{{ token }}" class="sf-toolbar" style="display: none"></div>
 {% include '@WebProfiler/Profiler/base_js.html.twig' %}
 <script>/*<![CDATA[*/
-    (function () {
+    document.addEventListener('DOMContentLoaded', function () {
```

Comment:
No need to care about old browsers here because this is a tool for developers (in the profiler for example we use flexbox that it's not supported in old browsers).